lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
I'm sure you've already been through this, but it's confusing why the existing help wouldn't work here?